crml/staking/src/tests.rs

```diff
@@ -208,45 +207,31 @@ fn rewards_should_work() {
 			},
 		);
 
-		<Payee<Test>>::insert(&2, RewardDestination::Stash);
-		assert_eq!(Staking::payee(2), RewardDestination::Stash);
-		assert_eq!(Staking::payee(11), RewardDestination::Account(10));
+		assert_eq!(Staking::payee(&2), RewardDestination::Stash);
+		assert_eq!(Staking::payee(&11), RewardDestination::Controller);
+
+		Rewards::new_fiscal_era();
```

Comment:
this would be preferred as a unit test, fiscal era start should be a by product
```suggestion
		start_era(0)
```

Reply:
start_era(0) couldn't do the job because the era 0 and session 0 is already configured and thus start_era(0) does nothing. I moved the very first new_fiscal_era to the genesis mocker for the tests instead.